docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -27,10 +30,9 @@ providing parameters via a config file, which simplifies node deployment.
 
 The subset of the Kubelet's configuration that can be configured via a file
 is defined by the `KubeletConfiguration` struct
-[here (v1alpha1)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go).
+[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go).
 The configuration file must be a JSON or YAML representation of the parameters
```

Comment:
It'd be good to provide an example of the config file. Then users/administrators can write their own config file easily. @mtaufen WDYT?

Reply:
From a user's perspective, I'm always not confident enough to write such files without an example :)

Reply:
I agree, we should have an example. Should not block this PR though. I can write an example on Monday when I'm back at work (OOO next two days).

Reply:
^ I can add a small example tonight, and a more complete one later

Reply:
Added a very small example file.